### ISSUE TITLE
Add HEIC support

### ISSUE THREAD

**Issue by dbalzan** [bricelam/ImageResizer#143]
Many photos are now in HEIC format.  Can support for this format be added?

**Comment by racas999** [comment]
I`m trying to use it to resize HECI images but it returns a error "object reference not defined for a instance of object"

**Comment by JNavas2** [comment]
@racas999 
1. Have you read the entire thread?
2. Which version of Windows are you using?
3. Do you have a HECI codec installed?

**Comment by racas999** [comment]
Yes, I’ve read all thread; I use Windows 10 and I have HEIC codec installed


Cassio

P.S.: Enviado pelo meu celular; desculpe eventuais erros de digitação.

Esta mensagem é confidencial e se destina exclusivamente aos destinatários.

________________________________
De: John Navas <notifications@github.com>
Enviado: sábado, fevereiro 15, 2020 15:23
Para: bricelam/ImageResizer
Cc: racas999; Mention
Assunto: Re: [bricelam/ImageResizer] Add HEIC support (#143)


@racas999<https://github.com/racas999>

  1.  Have you read the entire thread?
  2.  Which version of Windows are you using?
  3.  Do you have a HECI codec installed?

**Comment by JNavas2** [comment]
And can you open these images in Photos?
If not, see [this article](https://www.windowscentral.com/how-open-heic-and-hevc-files-windows-10s-photos-app).

**Comment by racas999** [comment]
Yes. I can open and edit them in Photos.

 

De: John Navas <notifications@github.com> 
Enviada em: sábado, 15 de fevereiro de 2020 23:28
Para: bricelam/ImageResizer <ImageResizer@noreply.github.com>
Cc: racas999 <racas999@gmail.com>; Mention <mention@noreply.github.com>
Assunto: Re: [bricelam/ImageResizer] Add HEIC support (#143)

 

And can you open these images in Photos?
If not, see this article <https://www.windowscentral.com/how-open-heic-and-hevc-files-windows-10s-photos-app> .

**Comment by JNavas2** [comment]
And ImageResizer works fine on other kinds of images in the same folder?
Which version of ImageResizer?

**Comment by racas999** [comment]
Yes. It works fine with jpg images in the same folder.

 

Version 3.1.2 I think I`m using the latest since I just installed it.

 

De: John Navas <notifications@github.com> 
Enviada em: sábado, 15 de fevereiro de 2020 23:35
Para: bricelam/ImageResizer <ImageResizer@noreply.github.com>
Cc: racas999 <racas999@gmail.com>; Mention <mention@noreply.github.com>
Assunto: Re: [bricelam/ImageResizer] Add HEIC support (#143)

 

And ImageResizer works fine on other kinds of images in the same folder?
Which version of ImageResizer?

**Comment by JNavas2** [comment]
And have you tried [CopyTrans HEIC Codec for Windows](https://codecpack.co/download/CopyTrans-HEIC-Codec.html)?
